sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_models.py

```diff
@@ -211,6 +237,59 @@ def managed(self):
         return self._managed
 
 
+class KeyReleaseParameters(object):
+    """The parameters for key release.
+
+    :param str environment: The target environment assertion.
+    :keyword str algorithm: The encryption algorithm to use to protect the released key material. Possible values
+     include: "CKM_RSA_AES_KEY_WRAP", "RSA_AES_KEY_WRAP_256", "RSA_AES_KEY_WRAP_384".
+    :keyword str nonce: A client-provided nonce for freshness.
+    """
+
+    def __init__(self, environment, **kwargs):
+        # type: (str, **Any) -> None
+        self.environment = environment
+        self.algorithm = kwargs.get("algorithm", None)
+        self.nonce = kwargs.get("nonce", None)
+
+
+class KeyReleasePolicy(object):
+    """A key release policy.
+
+    :param data: Blob encoding the policy rules under which the key can be released.
+    :type data: bytes
+    :keyword content_type: Content type and version of key release policy. Defaults to 
+     'application/json; charset=utf-8; version=1.0'.
+    :type content_type: str
+    """
+
+    def __init__(self, data, **kwargs):
+        # type: (bytes, **Any) -> None
+        self.data = data
+        self.content_type = kwargs.get("content_type", None) or "application/json; charset=utf-8; version=1.0"
```

Comment:
"; version=1.0" should be removed (recent PR that hasn't merged yet). Though, we really shouldn't code defaults from the swagger. The service should use its defaults if not serialized and sent to the service. As written now, this creates a tight coupling we want to avoid.